fluent-bit/fluent-bit-go-loki/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.12 as build
```

Comment:
should this go under the `./cmd` subtree?

Reply:
I'm not against the idea 🤷‍♂

Reply:
I moved it there.